images/pr-creator/Dockerfile

```
@@ -15,10 +15,13 @@ RUN set -x && \
 RUN set -x && \
     git clone https://github.com/kubevirt/project-infra.git && \
     cd ./project-infra && \
-    cp ./hack/git-askpass.sh hack/git-pr.sh /usr/local/bin && \
+    cp ./hack/git-askpass.sh /usr/local/bin && \
+    cp ./hack/git-pr.sh /usr/local/bin && \
     chmod +x /usr/local/bin/git-askpass.sh /usr/local/bin/git-pr.sh && \
-    bazel build //robots/cmd/labels-checker:labels-checker && \
-    cp ./bazel-out/k8-fastbuild/bin/robots/cmd/labels-checker/labels-checker_/labels-checker /usr/local/bin && \
+    cd robots/cmd/labels-checker && \
+    /usr/local/bin/runner.sh /bin/sh -ce 'go build -o labels-checker' && \
```

Can you build with something like 
```suggestion
    /usr/local/bin/runner.sh /bin/sh -ce 'go build ./robots/cmd/labels-checker' && \
```
that would allow us to remove the two `cd` lines.

---

@brianmcarey Thanks for the feedback.
I've addressed this replacing all code incl. and inbetween the two cd lines as below and tested.
`/usr/local/bin/runner.sh /bin/sh -ce 'go build -o /usr/local/bin/labels-checker ./robots/cmd/labels-checker' && \`